src/main/core/logger/log_wrapper.c

```diff
@@ -12,7 +12,7 @@ static void _flush(Logger* logger) { rustlogger_flush(); }
 
 static bool _isEnabled(Logger* logger, LogLevel level) { return rustlogger_isEnabled(level); }
 
-static void _setLevel(Logger* logger, LogLevel level) { rustlogger_setLevel(level); }
+static void _setLevel(Logger* logger, LogLevel level) { warning("Setting the log level is not supported"); }
```

Comment:
Remove?

Reply:
The `logger_setLevel` is still needed for the shim, so we can't remove `setLevel` from `Logger`. Since the shim and shadow use the same `Logger` interface, we need to set this `setLevel` function to something in shadow, so it's set to this dummy `_setLevel()` function. I don't see a nice way of removing this function without decoupling the shim and shadow loggers.